Cargo.toml

```diff
@@ -14,6 +14,8 @@ members = [
 	"chains/orchestrator-relays/node/*",
 	"chains/orchestrator-relays/runtime/dancelight",
 	"chains/orchestrator-relays/runtime/dancelight/constants",
+	"chains/orchestrator-relays/runtime/starlight",
```

Comment:
I believe starlight runtime wasm is not being built because we don't depend on it in `tanssi-relay`. I guess that's fine, and we will add it there in a future PR? Because we need that for typescript tests and zombienet tests anyway.

Reply:
Let me figure it out in this PR. Because guys will work on TS tests based on this PR.

Reply:
Will be added in a separate PR ([details](https://moondance-labs.slack.com/archives/C07HF2C5QJ1/p1745495855785099?thread_ts=1744368447.805829&cid=C07HF2C5QJ1)).